src/main/java/com/zendesk/maxwell/MaxwellTableCache.java

```diff
@@ -22,7 +23,7 @@ public void processEvent(Schema schema, MaxwellFilter filter, TableMapEvent even
 
 			Table tbl = db.findTable(tblName);
 
-			if ( filter != null && filter.isTableBlacklisted(tblName) )
+			if ( filter != null && filter.isTableBlacklisted(tbl.getDatabase().getName(), tblName) )
```

Comment:
that will handle the case where the table has not been tracked inside the database, but not the case where the database has not been tracked